### DISCUSSION TITLE
Include immediate package rebuild instructions in patching example

### DISCUSSION THREAD

**Discussion by W4RH4WK** [microsoft/vcpkg#39512] [General]
I've come across vcpkg's [patching example](https://learn.microsoft.com/en-us/vcpkg/examples/patching), which is already quite helpful.
However, I think there's one important step missing here:

The user should build and install the package from its modified source before the patch is created. Ideally I don't want to go through the hassle of creating a patch just to try out a modification. Creating the patch should be the last step of this development workflow.

Edit: I just realized that https://github.com/microsoft/vcpkg/issues/16874 is a major blocker for the general case. Manifest mode, while being the recommended mode used for development, does not support editing and rebuilding dependencies.

While the patching example should still be extended, there should be an easy way to develop patches for a dependency that is used by a project.

**Comment by autoantwort** [comment]
Simply use `--editable`?

**Reply by W4RH4WK** [reply]
> Simply use `--editable`?

What do you mean? The example already uses `--editable`, which also does not work in manifest mode as outline above.
